CMakeLists.txt

```diff
@@ -81,6 +81,7 @@ if(APPLE)
 	set_property(DIRECTORY
 		APPEND
 		PROPERTY COMPILE_DEFINITIONS SC_DARWIN)
+	set(CMAKE_INSTALL_PREFIX "${CMAKE_BINARY_DIR}/Install")
```

Comment:
ok ... not having self-contained app bundles is a shortcoming of sc's cmake implementation, though.